config/settings/testing.py

```diff
@@ -98,3 +101,22 @@ def __call__(self, _, strict):
         'DB': 0
     }
 }
+
+
+if os.getenv('GITHUB_WORKFLOW'):
+    DATABASES = {
```

Comment:
This configuration is needed for the connection of MySQL with the backend in the github actions workflow. As this is set only for the github actions environment, I think it is fine to define it like this.
WDYT?

Reply:
But the configuration is already definied before to running the tests. When you run `manage` with `test` it takes the configuration from the file under the testing section.

Reply:
Yeah, I understand that. I didn't want to change the actual configuration which might affect the current users, but it makes sense that it can always be changed. I'm removing this extra configuration and updating the existing one with the password and other required things.